docs/guides/endgame.md

```diff
@@ -158,20 +174,25 @@ Checklist:
 
 ### Week 2: Project work
 
-No items for endgame team to complete during this time.
+Checklist:
+
+- [ ] Patch release on Monday
 
 ### Week 3: Project work
 
 Checklist:
 
-- [ ] Schedule mid-project playbacks for afternoon Thursday
+- [ ] Patch release on Monday
+- [ ] Mid-project playbacks on Thursday afternoon
```

Comment:
Any reason to change the language from "schedule" as that's the action item?

Reply:
@joshblack I added a note in the PR changelog about updating the scheduling of meetings to be done in the first week so everyone just knows ahead of time for a few reasons: we won't risk any possible double scheduling of meetings if people just have those in their calendar before the week of, more clear/transparent expectations of timeline for the release, in the past it seems like it usually works out fine to schedule these meetings at the beginning of the release cycle. So the idea was that this meeting would already be scheduled and end game leader just has to run it on week 3.